angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.component.ts

```diff
@@ -63,9 +63,7 @@ import { ISprkMastheadSelectorChoice } from '../sprk-masthead-selector/sprk-mast
             <sprk-icon
               [iconName]="triggerIconName"
               additionalClasses="
-                sprk-c-Icon--filled-current-color
-                sprk-c-Icon--stroke-current-color
-                sprk-u-mhs
+                sprk-c-Icon
```

Comment:
hey @yodasw16 , the sprk-icon component should be applying the sprk icon class to this already so shouldnt need to add it. Is it not working or was this a typo?

Reply:
Oh, hmm. I didn't test without it. The React version was working correctly, so I matched the others with it. Let me try.